tensorflow/tools/docker/README.md

```diff
@@ -59,6 +59,13 @@ Building TensorFlow Docker containers should be done through the
 script. The raw Dockerfiles should not be used directly as they contain strings
 to be replaced by the script during the build.
 
+Attempting to run [parameterized_docker_build.sh] from a binary docker image such as for example tensorflow/tensorflow:latest will not work. One needs to execute the script from a developer docker image since by contrast with a binary docker image it contains not only the compiled solution but also the tensorflow source code. Please select the appropriate developer docker image of tensorflow at tensorflow/tensorflow:[right tag](https://hub.docker.com/r/tensorflow/tensorflow/tags/).
+
+The smallest command line to generate a docker image will then be:
+docker run -it tensorflow/tensorflow:right tag
```

Comment:
Wrap the whole line inside "```" block
also replace "right tag" with <tag>

Reply:
Replaced "right tag" with "right_tag" is that what you meant?

Reply:
Looks like I meant to say more, but I completely forgot what I wanted to say here :)